shell/pom.xml

```diff
@@ -69,5 +69,45 @@
       <artifactId>alluxio-core-common</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
I added dependencies to UFS so the UnderFileSystemFactoryRegistry can discover all the UFS  factories. Is there a way to make that work without adding dependencies?